src/cmd/cmd.cpp

```diff
@@ -167,11 +167,40 @@ void sync(const SyncCTX &ctx)
             }
         }
     });
+    QObject::connect(engine, &SyncEngine::aboutToRemoveAllFiles, engine, [ctx](OCC::SyncFileItem::Direction dir, std::function<void(bool)> callback) {
```

Comment:
Can we rename `callback` to `cancel` or `cancelCallback` or `stop` or something like that?

My brain has a hard time parsing what true and false arguments mean with `callback`...